### PR TITLE
dumbpad v3.x - hotswap and rgb

### PR DESCRIPTION
**Pull request by deveth0**
Here we go... New revision of the dumbpad featuring Kailh Hotswap Sockets and (optional) RGB per key. 

Although this is the first revision using SMD components, it's still quite easy to solder, as the components can be fixed to the PCB first and no really small parts need to be soldered.

Note: i have not published the QMK files yet, still working on those. I'll try to get them into the qmk upstream repo asap.

### CODE REVIEW

**Comment by deveth0** [comment]
and the missing QMK PR: https://github.com/qmk/qmk_firmware/pull/16285

**Comment by imchipwood** [comment]
Sorry I didn't get to this yesterday - I've reviewed a little of it so far and will try to finish after work tonight!

File: hotswap_rgb/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Submit either the KiCad `.kicad_pcb` or the gerber files to your prefered PCB manufacturer.
+
+### Notes on soldering
+
+It's recommended to start with the Sockets and then add the remaining parts.
+
+Solder the SK6812 LEDs with the missing corners aligned with the corners printed on the PCB as shown in the image.
+
+*NOTE:* the orientation of the LEDs changes between the lines!
```
Comment (imchipwood) [inline hotswap_rgb/README.md:49]: Can you replace the photo with one that shows LEDs on rows with different orientations? Specifically, unsoldered LEDs placed in the proper orientation.

Maybe even a big red circle around the chamfered edge :P

File: hotswap_rgb/fp-lib-table [hunk 2]
```diff
@@ -0,0 +1,4 @@
+(fp_lib_table
+  (lib (name "dumbpad")(type "KiCad")(uri "$(KIPRJMOD)/dumbpad.pretty")(options "")(descr ""))
+  (lib (name "Library")(type "KiCad")(uri "D:/Develop/KiCad/dumbpad_v2/Library.pretty")(options "")(descr ""))
```
Comment (imchipwood) [inline hotswap_rgb/fp-lib-table]: I'm unfamiliar with KiCad - this looks like a local library import. Is this library standard with KiCad or custom? If it's custom, please include it in the PR.

**Review by imchipwood — CHANGES_REQUESTED** [review]
Couple things:
1. Please include a zip directory with all the required gerber files for submitting jobs to PCB manufacturers (look at the `combo` folder for an example)
2. Please move the images to an inner folder, or consider hosting them externally (imgur, etc.)

Other than that, this looks great! I already ordered hotswap sockets and LEDs so I can build some :)

Thanks!
Chip

File: hotswap_rgb/fp-lib-table (hunk 2 above)
Comment (deveth0) [reply hotswap_rgb/fp-lib-table]: ahh ffs, sorry, my fault.

**Comment by deveth0** [comment]
1, [hotswap_rgb_gerber.zip](https://github.com/deveth0/dumbpad/blob/master/hotswap_rgb/hotswap_rgb_gerber.zip) ? 
2, should i also move the other images (e.g. combo or reversible/dumbpad.png) to separate folders?

File: hotswap_rgb/README.md (hunk 1 above)
Comment (deveth0) [reply hotswap_rgb/README.md:49]: something like this? https://github.com/deveth0/dumbpad/blob/master/hotswap_rgb/img/dumbpad_v3_1_rgb_mounting.jpg

**Comment by imchipwood** [comment]
1. yep!
2. Just the photos you've added, since there are multiple (other folders only have one photo - just keeping things tidy!)

File: hotswap_rgb/README.md (hunk 1 above)
Comment (imchipwood) [reply hotswap_rgb/README.md:49]: perfect!

File: hotswap_rgb/fp-lib-table (hunk 2 above)
Comment (deveth0) [reply hotswap_rgb/fp-lib-table]: here we go, finally ...

**Review by imchipwood — APPROVED** [review]

**Pull request merged**